packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -17,7 +17,13 @@ const allOptions: NodeVersion[] = [
 
 const supportedOptions = allOptions.filter(o => !isDiscontinued(o));
 const pleaseUse =
-  'Please use one of the following supported ranges in your `package.json`: ';
+  'Please use one of the following `engines` in your `package.json`: ' +
```

Comment:
Let's get rid of this special case, let's always show the message below - it's much more clear

Reply:
Thanks, I updated the message to always say 

```
Please set "engines": { "node": "12.x" } in your `package.json` file to upgrade to Node.js 12
```